apis/validator/liveness.yaml

```diff
@@ -0,0 +1,61 @@
+post:
+  tags:
+    - Validator
+  summary: "Indicates if a validator has been observed on the network"
+  operationId: "getLiveness"
+  description: "Requests the beacon node to indicate if a validator has been
+    observed to be live a given epoch. The beacon node might detect liveness by
+    observing messages from the validator on the network, in the beacon chain,
+    from its API or from any other source. It is important to note that the
+    values returned by the beacon node are not canonical; they are best-effort
+    and based upon a subjective view of the network."
+  requestBody:
+    description: "An array of the validator indices for which to detect liveness."
+    required: true
+    content:
+      application/json:
+        schema:
+          title: GetAttesterDutiesBody
+          type: object
+          properties:
+            epoch:
+              $ref: "../../../beacon-node-oapi.yaml#/components/schemas/Uint64"
```

Comment:
I think you'll find references in this file are only `../../beacon-node-oapi.yaml` and that's probably the main build issue...